client/views/msg_condensed.tpl

```diff
@@ -0,0 +1,7 @@
+<div class="msg {{type}} closed" data-time="{{time}}">
+	<span class="time hide-text">{{tz time}}</span>
+	<span class="from"><span class="expander"></span></span>
```

Comment:
Anyone with strong arguments against having this on the right of the text, to make it consistent with preview toggles? I.e. instead of:

<img width="169" alt="screen shot 2017-07-31 at 00 48 36" src="https://user-images.githubusercontent.com/113730/28763180-27f857e0-758a-11e7-8a0d-44f63d0b48be.png">

We would have:

<img width="169" alt="screen shot 2017-07-31 at 00 48 25" src="https://user-images.githubusercontent.com/113730/28763185-2e2aba90-758a-11e7-8933-c7b8a40f9d44.png">

I understand that having no icons on the left can be confusing since this is not a regular message, but it's odd to have it on the left of the column separator, knowing it is clickable. We could (but don't have to) have an icon on the left, if so, not sure which one, but it could be any of:
- http://fontawesome.io/icon/bolt/
- http://fontawesome.io/icon/question/
- http://fontawesome.io/icon/asterisk/
- http://fontawesome.io/icon/info/
- http://fontawesome.io/icon/certificate/
- ...

I honestly don't have a preference myself, I'm not sure we need an icon anyway, so I'm open for suggestions.